src/views/Timeline.vue

```diff
@@ -11,36 +11,39 @@ import { useProfiles } from '@/composables/useProfiles';
 import { useFollowSpace } from '@/composables/useFollowSpace';
 import { useWeb3 } from '@/composables/useWeb3';
 import verified from '@/../snapshot-spaces/spaces/verified.json';
+import zipObject from 'lodash/zipObject';
+import { useStore } from '@/composables/useStore';
 
-const verifiedSpaces = Object.entries(verified)
-  .filter(space => space[1] === 1)
-  .map(space => space[0]);
+const { store } = useStore();
 
-import zipObject from 'lodash/zipObject';
-const filterBy = ref('all');
 const loading = ref(false);
-const proposals = ref([]);
 
 const route = useRoute();
 const { followingSpaces, loadingFollows } = useFollowSpace();
 const { web3 } = useWeb3();
 
-const following = computed(() => {
-  return route.name === 'timeline' ? followingSpaces.value : verifiedSpaces;
+const spaces = computed(() => {
+  const verifiedSpaces = Object.entries(verified)
+    .filter(space => space[1] === 1)
+    .map(space => space[0]);
+  if (route.name === 'timeline') return followingSpaces.value;
+  if (route.name === 'explore') return verifiedSpaces;
+  else return [];
 });
 
-const isTimeline = computed(() => route.name === 'timeline');
-
-watch(following, () => {
-  proposals.value = [];
-  load();
+watch(spaces, () => {
+  if (route.name === 'timeline' || route.name === 'explore') {
+    store.timeline.proposals = [];
+    load();
+  }
 });
 
-const { loadBy, limit, loadingMore, stopLoadingMore, loadMore } =
-  useInfiniteLoader();
+const isTimeline = computed(() => route.name === 'timeline');
```

Comment:
Why not move this up a bit?

Then line 29/30 can be:

```js
return isTimeline ? followingSpaces.value : verifiedSpaces;
```

Reply:
Doesn't seem to work well considering that we will add more filters.